e2e/e2e_docker_test.go

```diff
@@ -39,7 +39,39 @@ func runPipelineTests(t *testing.T, path string) {
 		c := c // capture range variable
 		t.Run(c.Path, func(t *testing.T) {
 			t.Parallel()
-			r, err := runner.NewRunner(c, runner.CommandFnRender)
+			r, err := runner.NewRunner(c, c.Config.TestType)
+			if err != nil {
+				t.Fatalf("failed to create test runner: %s", err)
+			}
+			if r.Skip() {
+				t.Skip()
+			}
+			err = r.Run()
+			if err != nil {
+				t.Fatalf("failed when running test: %s", err)
+			}
+		})
+	}
+}
+
+func TestEval(t *testing.T) {
+	runPipelineTests(t, "./fn/eval")
```

Comment:
Little odd that `TestEval` is calling runPipelineTests ?

Reply:
I just notice that `runPipelineTests` and `runEvalTests` are exactly same. Will remove one.